tsp-cli-client

```diff
@@ -64,7 +64,7 @@ def print_time(elem):
         if ns > 1000:
             return '{0} µs'.format(ns / 1000)
         return '{0} ns'.format(ns)
-    except ValueError as e:
+    except ValueError:
         return elem
```

Comment:
can we log this instead?

Reply:
> can we log this instead?

Good suggestion; thanks.
This should be another commit though, as this very one focuses on fixing lint issues solely.
Indeed, 'e' is not used, hence the lint issue fixed herein.

Reply:
-Done; I amended this specific commit thus rebased its follow-ups.
(I also pushed some more small changes on top.)

Reply:
<=> https://github.com/theia-ide/tsp-python-client/compare/4e4de950c06cc6c9cc0096ae95702b1868188226..9667859e8247c4c1a8860a5f7d07cfac238e0005